libraries/debug-certificate-manager/src/CertificateManager.ts

```diff
@@ -70,20 +75,35 @@ interface ISubjectAltNameExtension {
   altNames: readonly IAltName[];
 }
 
-interface IAltName {
+/**
+ * Fields for a Subject Alternative Name of type DNS Name
+ */
+interface IDnsAltName {
   type: 2;
   value: string;
 }
+/**
+ * Fields for a Subject Alternative Name of type IP Address
+ */
+interface IIPAddressAltName {
+  type: 7;
+  ip: string;
```

Comment:
Why not make this `value` as well? Could avoid all the mapping below.

Reply:
Updated comments here. `node-forge` requires this to be named `ip`, otherwise it emits a malformed certificate.